vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -76,49 +74,51 @@ module InlineParameterNameHints =
             false
 
     let isUnionCaseValidForHint (symbol: FSharpUnionCase) (symbolUse: FSharpSymbolUse) =
-        symbolUse.IsFromUse && symbol.DisplayName <> "(::)"
+        symbolUse.IsFromUse
+        && symbol.DisplayName <> "(::)"
+        // If a case does not use field names, don't even bother getting applied argument ranges
+        && Seq.toList symbol.Fields |> Seq.exists doesFieldNameExist
```

Comment:
Curious - why converting sequence to list, and then still use `Seq.exists`? Shouldn't this be either without the convesion, or use `List.exists`?

Reply:
No reason, copypasting without thinking - thanks, fixed